### ISSUE TITLE
Clicking on Notifications leads to error page

### ISSUE THREAD

**Issue by erkage** [usebruno/bruno#3556]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

When I click the bottom left Notifications icon I got the following error page. Tried to restart the app, updated to the newest version v1.35.0 – prev was on v1.31.0.

### Screenshots/Live demo link

<img width="1728" alt="Screenshot 2024-11-27 at 10 08 25" src="https://github.com/user-attachments/assets/d4eae37c-59ac-45fc-ad2d-743b9ea06393">

**Comment by erkage** [comment]
Hmm, actually the `localStorage` is contains only the following:
```
{
    "bruno.theme": "\"dark\"",
    "bruno.installedOn": "2024-03-21T11:55:46.051Z",
    "bruno.anonymousTrackingId": "qxLKd7UA6G..."
}
```
Should I still clear it?

And despite that I have 1 unread Notifications:
<img width="218" alt="Screenshot 2024-11-27 at 13 51 24" src="https://github.com/user-attachments/assets/3d43070a-57ba-4b3f-986a-3bd761276737">

**Comment by helloanoop** [comment]
Just merged the PR.
The fix will be released in `v1.36.0` scheduled to go out on 4 Dec 2024

**Comment by erkage** [comment]
Hey @Pragadesh-45 , @helloanoop 

Just updated and checked the Notifications and it works. Thanks.

Have a nice day.
